public/app/plugins/datasource/prometheus/language_provider.ts

```diff
@@ -85,7 +95,7 @@ export default class PromQlLanguageProvider extends LanguageProvider {
     return PromqlSyntax;
   }
 
-  request = async (url: string) => {
+  request = async (url: string, defaultValue: any): Promise<any> => {
```

Comment:
The response for the new API is a dict, as opposed to a list for all other metadata queries. Had to introduce a mandatory default value to make sure caller is aware of return type by having to give a default value.

Reply:
We could make it
```
request = async <T = any>(url: string, defaultValue: T): Promise<T> => {
```
To make it easier to add optional type safety.